bigtable-autoscaler-service.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: bigtable-autoscaler
+  annotations:
+    cloud.google.com/neg: '{"ingress": true}'
+  labels:
+    app: bigtable-autoscaler
+spec:
+  # Required for Container Engine Ingress:
+  # https://github.com/kubernetes/ingress/tree/master/controllers/gce
+  type: NodePort
+  ports:
+    - port: 8080
+      targetPort: http
+      name: http
+  selector:
+    app: bigtable-autoscaler
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: bigtable-autoscaler 
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: bigtable-autoscaler 
+  template:
+    metadata:
+      labels:
+        app: bigtable-autoscaler 
+      annotations:
+        cluster-autoscaler.kubernetes.io/safe-to-evict: 'true'
+    spec:
+      containers:
+      - name: bigtable-autoscaler 
+        image: us.gcr.io/triggeredmail/bigtable-autoscaler:0.0.1 
```

Comment:
To use the deploy tool, I'd rather have `{{image}}` here.

Reply:
yeah forgot to add